### PR TITLE
Implementation of ideasboard

### PR DESCRIPTION
**Pull request by ddhanesha**
## Description

Resloves  #584 

## Todo

- [x] PR has a meaningful title?
- [x] Lint passes?
- [x] Ran `yarn format` in frontend?

### CODE REVIEW

**Comment by positiveimpact** [comment]
I think I would just make one PR for the whole ideas board. So I'l also work on this branch if you're fine with it :)

**Comment by ddhanesha** [comment]
> I think I would just make one PR for the whole ideas board. So I'l also work on this branch if you're fine with it :)

Yeah that’s fine.

File: backend/ideas/models/ideas.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        max_length=256
+    )
+
+    summary = models.CharField(
+        help_text="Summary of an idea",
+        verbose_name="Summary",
+        max_length=1024
+    )
+
+    description = models.TextField(
```
Comment (positiveimpact) [inline backend/ideas/models/ideas.py:25]: I think we'll just have a short description? but not sure if that is 100% decided yet

File: backend/ideas/models/translation.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    language = models.ForeignKey(
+        Language,
+        help_text="Points to the language idea is translated for",
+        verbose_name="Language",
+        related_name="idea_language",
+        on_delete=models.CASCADE
+    )
+
+    name = models.CharField(
```
Comment (positiveimpact) [inline backend/ideas/models/translation.py:24]: I would keep the convention of having `[fieldName]_translation` here, e.g. `name_translation` in this case

**Review by positiveimpact — COMMENTED** [review]
LGTM, added 2 minor comments. Will also start committing to this PR

File: backend/ideas/models/translation.py (hunk 2 above)
Comment (ddhanesha) [reply backend/ideas/models/translation.py]: Done

File: backend/ideas/models/ideas.py (hunk 1 above)
Comment (ddhanesha) [reply backend/ideas/models/ideas.py]: Removed

**Pull request merged**